<!DOCTYPE html>
<!--[if IE 8]> <html lang="en" class="ie8"> <![endif]-->
<!--[if IE 9]> <html lang="en" class="ie9"> <![endif]-->
<!--[if !IE]><!-->
<html lang="en">
  <!--<![endif]-->
  <head>
  <title>James A. Fellows Yates</title>

  <!-- Meta -->
  <meta charset="utf-8">
  
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="description" content="Personal website and CV for James A. Fellows Yates">

  <!-- Favicon -->
  <link rel="shortcut icon" href="favicon.ico">

  <!-- Global CSS -->
  <link rel="stylesheet" href="/assets/plugins/bootstrap/css/bootstrap.min.css">

  <!-- Plugins CSS -->
  <link rel="stylesheet" href="/assets/plugins/font-awesome/css/all.css">

  <!-- Theme CSS -->
  <link id="theme-style" rel="stylesheet" href="/assets/css/main.css">
  <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
  <!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
    <![endif]-->
</head>

  <body>
    <div class="wrapper">
      
<div class="sidebar-wrapper">

  <div class="profile-container">

    
    <img class="avatar" src="/assets/images/profile.png" alt="profile picture" />
    

    
    <h1 class="name">Dr. James A. Fellows Yates</h1>
    

    
    <h3 class="tagline">Post Doctoral Bioinformaician</h3>
    

  </div><!--//profile-container-->

  <div class="contact-container container-block">
  <ul class="list-unstyled contact-list">

    
    <li class="email"><i class="fas fa-envelope"></i>
      <a href="mailto:jfy133@gmail.com">jfy133@gmail.com</a>
    </li>
    

    

    

    
    <li class="timezone"><i class="fas fa-clock"></i>
      <a href="">Central European Time</a>
    </li>
    

    

    
    <li class="website"><i class="fas fa-globe-americas"></i>
      <a href="http://jafy.eu" target="_blank">jafy.eu</a>
    </li>
    

    
    <li class="linkedin"><i class="fab fa-linkedin"></i>
      <a href="https://linkedin.com/in/james-fellows-yates-999859179" target="_blank">james-fellows-yates-999859179</a>
    </li>
    

    

    
    <li class="github"><i class="fab fa-github"></i>
      <a href="http://github.com/jfy133" target="_blank">jfy133</a>
    </li>
    

    

    

    
    <li class="twitter"><i class="fab fa-twitter"></i>
      <a href="https://twitter.com/@jfy133" target="_blank">@jfy133</a>
    </li>
    

    

    

    

    

  </ul>
</div><!--//contact-container-->


  
    

  
  <div class="education-container container-block">

    <h2 class="container-block-title">
      Education
    </h2>

    
    <div class="item">
      <h4 class="degree">PhD in Bioinformatics</h4>
      <h5 class="meta">Friedrich-Schiller-UniversitÃ¤t Jena (Max Planck Institute for the Science of Human History / Max Planck Institute for Evolutionary Anthropology, ğŸ‡©ğŸ‡ª)</h5>
      <div class="time">2015 - 2022</div>
    </div><!--//item-->
    
    <div class="item">
      <h4 class="degree">MSc in Naturwissenschaftliches ArchÃ¤ologie</h4>
      <h5 class="meta">Eberhard-Karls-UniversitÃ¤t TÃ¼bingen (ğŸ‡©ğŸ‡ª)</h5>
      <div class="time">2013 - 2015</div>
    </div><!--//item-->
    
    <div class="item">
      <h4 class="degree">BSc (Hons) in Biomolecular Archaeology</h4>
      <h5 class="meta">University of York (ğŸ‡¬ğŸ‡§)</h5>
      <div class="time">2010 - 2013</div>
    </div><!--//item-->
    

  </div><!--//education-container-->
  


  

  

<div class="languages-container container-block">

  <h2 class="container-block-title">
    Languages
  </h2>

  <ul class="list-unstyled interests-list">
    
    <li>
      English
      <span class="lang-desc">(Native)</span>
    </li>
    
    <li>
      German
      <span class="lang-desc">(B2)</span>
    </li>
    
    <li>
      Spanish
      <span class="lang-desc">(Basic)</span>
    </li>
    
    <li>
      Catalan
      <span class="lang-desc">(Basic)</span>
    </li>
    
  </ul>

</div><!--//interests-->


  



  
    <div class="remove-container container-block">
  <h2 class="container-block-title">
    About Theme
  </h2>
  <ul class="list-unstyled interests-list">
    <li><a target="_blank" href="https://www.youtube.com/watch?v=Jnmj1dXDbNk">How to use?</a></li>
     <li><a class="github-button" href="https://github.com/sharu725/online-cv" data-icon="octicon-star" data-show-count="true" aria-label="Star sharu725/online-cv on GitHub">Star</a>
     <script async defer src="https://buttons.github.io/buttons.js"></script>
  </ul>
</div><!--//About-->
  

</div><!--//sidebar-wrapper-->


      <div class="main-wrapper"><h1 id="sequin-submission-to-genbank-tutorial">Sequin Submission to Genbank Tutorial</h1>

<h3 id="james-a-fellows-yates">James A. Fellows Yates</h3>

<p>As I came to the end of my M.Sc. and writing the corresponding journal article,
one thing I was aware of was that all previous papers on woolly mammoth
mitchondrial genomes to NCBIâ€™s GenBank database. To continue with this
established â€˜traditionâ€™, I decided to investigate how to do this. Worringly, most people I spoke to didnâ€™t have very positive views on this
procdure, saying that it was complicated, error prone and badly documented and
that it wasnâ€™t worth my time. In fact, with a bit of help from the authors of
some of the previous papers and with NCBI itself, the overall task ended up
being easier than expected (admittedly after a considerable bit of trial and
error).</p>

<p>Here, I describe the eventual steps that I did to (successfully!) annotate a
set of mtGenomes and make a submission to GenBank. However, note that these
instructions are only valid for this use-case, and would not be necessarily
valid for other situations.</p>

<p>In this case, we assume that you have multiple
sequences of a species that already has an annotated reference sequence on
GenBank. I use here the â€˜Krauseâ€™ reference (Accession: NC_007596.2) and my
reconstructed mtGenomes.</p>

<h2 id="preparation">Preparation</h2>

<ol>
  <li>Download the program <code class="language-plaintext highlighter-rouge">Sequin</code> from <a href="https://www.ncbi.nlm.nih.gov/Sequin/">here</a> (If running Ubuntu 16.04 might have to install libxp6 from the PPA <a href="https://bugs.launchpad.net/ubuntu/+source/libxp/+bug/1517884/comments/11">here</a>).</li>
  <li>Download the reference sequence from Genbank in FASTA format and the FeatureTable format.</li>
  <li>Using a multisequence aligner tool like <a href="http://www.megasoftware.net/">MEGA</a> or <a href="http://www.geneious.com/">Geneious</a>, align your sequences to the reference, and export the file as a FASTA file (make sure to export gaps as Ns, and not â€˜-â€˜ as in the documentation, and keep the reference sequence!).</li>
  <li>
    <p>To save time later, you now want to change the FASTA headers for each sequnce to the NCBI format, adding as much information as you can using the â€˜modifiersâ€™ listed <a href="https://www.ncbi.nlm.nih.gov/Sequin/modifiers.html">here</a>. An example of one of my headers is as so:</p>

    <div class="language-bash highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="o">&gt;</span>JK2760 <span class="o">[</span><span class="nv">organism</span><span class="o">=</span>Mammuthus primigenius] <span class="o">[</span><span class="nv">country</span><span class="o">=</span>Germany] <span class="o">[</span><span class="nv">isolate</span><span class="o">=</span>JK2760] Mammuthus primigenius isolate JK2760 mitochondrion, partial genome
</code></pre></div>    </div>
  </li>
</ol>

<p>For the reference sequence you should also add the tag <code class="language-plaintext highlighter-rouge">[acc=NC_007596.2]</code>.</p>

<h2 id="using-sequin">Using Sequin</h2>

<ol>
  <li>Next we can start <code class="language-plaintext highlighter-rouge">Sequin</code>, select â€˜NCBI GenBankâ€™ and â€˜Start new submissionâ€™.</li>
</ol>

<p><img src="/assets/images/1-load_sequin.png" width="100%" /></p>

<ol>
  <li>Set the release date of the data (for example if you want to release on the same day as your paper is published - note you can update this later) and add a tentative title for the study.</li>
</ol>

<p><img src="/assets/images/2-submission_info_1.png" width="100%" /></p>

<ol>
  <li>Set you name, phone and email (fax can be skipped).</li>
</ol>

<p><img src="/assets/images/3-submission_info_2.png" width="100%" /></p>

<ol>
  <li>Enter the names of the authors of the study.</li>
</ol>

<p><img src="/assets/images/4-submission_info_3.png" width="100%" /></p>

<ol>
  <li>Enter the submitting institution address (fax can be skipped).</li>
</ol>

<p><img src="/assets/images/5-submission_info_4.png" width="100%" /></p>

<ol>
  <li>
    <p>Select the â€˜use normal submission dialogâ€™ option.</p>

    <p><img src="/assets/images/6-start_submission_dialog.png" width="100%" /></p>
  </li>
  <li>
    <p>Select â€˜Population Studyâ€™, â€˜Alignmentâ€™ and â€˜Original submissionâ€™.</p>

    <p><img src="/assets/images/7-submission_type.png" width="100%" /></p>
  </li>
  <li>
    <p>Now you can import your FASTA alignment through the â€˜Import Alignmentâ€™ button and select the following options:</p>

    <ul>
      <li>Trim Ns at the end of sequences: yes.</li>
      <li>Specify molecule: â€˜Genomic DNAâ€™ (apply to all).</li>
      <li>Specify topology: â€˜linearâ€™ (apply to all).</li>
    </ul>

    <p><img src="/assets/images/8-import_alignment.png" width="100%" />
<img src="/assets/images/9-set_molecule_information.png" width="100%" /></p>
  </li>
  <li>
    <p>Select â€˜Illuminaâ€™, â€˜assembled sequencesâ€™ and input your assembly program (in my case it is <code class="language-plaintext highlighter-rouge">bwa</code> and version â€˜0.7â€™).</p>

    <p><img src="/assets/images/10-set_sequencing_information.png" width="100%" /></p>
  </li>
  <li>
    <p>Next you can add various information like organisms, locations (e.g. here I selected mitochondrion) and genetic codes (to apply to all you can select the header). You can keep the genetic code as â€˜standardâ€™ or change to â€˜Vertebrate Mitochondrialâ€™. Some of this will have already been imported based on your FASTA headers.</p>

    <p><img src="/assets/images/11-add_sequence_metadata_1.png" width="100%" />
<img src="/assets/images/12-add_sequence_metadata_2.png" width="100%" /></p>
  </li>
  <li>You can skip the protein step.</li>
  <li>
    <p>You can also skip annotation (select â€˜noneâ€™) and â€˜Continue to record viewerâ€™.</p>

    <p><img src="/assets/images/13-skip_protein_annotation.png" width="100%" />
<img src="/assets/images/14-record_viewer.png" width="100%" /></p>
  </li>
  <li>
    <p>First, it is best to annotate any runs of Ns as sequencing gaps (as per advice from the NCBI helpdesk). You can do this by doing the following:</p>

    <ul>
      <li>In record viewer Go to Search Validate &gt; Validate.</li>
      <li>
        <p>Double click on a â€˜run of Nsâ€™ error.</p>

        <p><img src="/assets/images/15-validation_window.png" width="100%" /></p>
      </li>
      <li>In the sequence viewer, type in the start position (-1) in the â€˜Go toâ€™ box.</li>
      <li>
        <p>Then click and drag until all the Ns are selected.</p>

        <p><img src="/assets/images/16-add_sequence_gap_1.png" width="100%" /></p>
      </li>
      <li>
        <p>Then go to Features &gt; Remaining Features &gt; misc_features and under the â€˜Propertiesâ€™ and then â€˜Commentâ€™ tab you can add â€˜sequencing gap, estimated length 119 bpâ€™.</p>

        <p><img src="/assets/images/17-add_sequence_gap_2.png" width="100%" /></p>
      </li>
    </ul>
  </li>
  <li>Once all of those are done, in the Record Viewer you need to select from the drop down menu the reference sequence (in this case NC_007596.2) and then File &gt; Open the Genbank (flat)file or the feature table. This should automatically load all the annotations of the original sequence, also â€˜importingâ€™ features that in the latest version of Sequin fail due to conflicts (e.g. I could not manually make a <code class="language-plaintext highlighter-rouge">/trans_exep</code>, but when I imported it gives an error but keeps the feature).</li>
  <li>
    <p>Finally in the Record Viewer you can go Edit &gt; Feature Propagate. This will copy over all of the annotations on the reference sequnce to your other sequences, but doing the correct DNA to amino acid translation.</p>

    <p><img src="/assets/images/18-propagate_features.png" width="100%" /></p>
  </li>
  <li>You should now do another â€˜validateâ€™ to find any errors. For example a couple of my translated amino acid sequences began with XXXXXX, so the <code class="language-plaintext highlighter-rouge">Sequin</code> couldnâ€™t find the start codon and thus failed. I just removed these feature entries (and informed NCBI staff).</li>
  <li>Once you have no failures, you can go in Record Viewer to â€˜File &gt; Prepare Submissionâ€™ and send the corresponding file to NCBI genbank as per the instructions. <a href="https://www.ncbi.nlm.nih.gov/books/NBK53709/#gbankquickstart.Submission_using_Sequin">here</a>.</li>
</ol>

<p>Once done, after a few days you should recieve a message from the NCBI either with
corrections to be made or a list of accession numbers, which will go live on
the date you selected in Sequin.</p>

<p>To update or to get the release date changed you can just email them again at
the same address (you donâ€™t need to do a resubmission or use sequin).</p>
</div>
    </div>

    <footer class="footer">
    <div class="text-center">
        <!--/* This template is released under the Creative Commons Attribution 3.0 License. Please keep the attribution link below when using for your own project. Thank you for your support. :) If you'd like to use the template without the attribution, you can check out other license options via our website: themes.3rdwavemedia.com */-->
        <small class="copyright">Designed with <i class="fas fa-heart"></i> by <a href="http://themes.3rdwavemedia.com" target="_blank" rel="nofollow">Xiaoying Riley</a>. Contents Â©ï¸ James A. Fellows Yates, 2022
</small>
    </div><!--//container-->
</footer><!--//footer-->
 <!-- Javascript -->
<script
  type="text/javascript"
  src="/assets/plugins/jquery-1.11.3.min.js"
></script>
<script
  type="text/javascript"
  src="/assets/plugins/bootstrap/js/bootstrap.min.js"
></script>

<!-- custom js -->
<script
  type="text/javascript"
  src="/assets/js/main.js"
></script>

<!--  -->

  </body>
</html>
